autogpt_platform/backend/backend/data/invited_user.py

```
@@ -463,50 +473,90 @@ async def bulk_create_invited_users_from_file(
-            results.append(
-                BulkInvitedUserRowResult(
-                    row_number=row.row_number,
-                    email=normalized_email,
-                    name=row_name,
-                    status="CREATED",
-                    message="Invite created",
-                    invited_user=invited_user,
+            continue
+
+    # Second pass: create invites in batches to bound concurrent DB load.
+    for i in range(0, len(valid_rows), _BULK_INVITE_BATCH_SIZE):
```

Ruff (0.15.6)

[error] Undefined name `_BULK_INVITE_BATCH_SIZE`

(F821)